### PR TITLE
Remove completed TODOs

### PR DESCRIPTION
**Pull request by jla524**

### CODE REVIEW

**Comment by geohot** [comment]
Huh, how did a test fail?

**Comment by jla524** [comment]
> Huh, how did a test fail?

not sure why matmul was causing nans, but I couldn't reproduce it on my device

File: test/models/test_mnist.py [hunk 1]
```diff
@@ -94,13 +94,13 @@ def test_conv(self):
...
     assert evaluate(model, X_test, Y_test) > 0.94   # torch gets 0.9415 sometimes
 
-  @unittest.skip("slow and training batchnorm is broken")
+  @unittest.skip("training is slow")
   def test_conv_with_bn(self):
     np.random.seed(1337)
     model = TinyConvNet(has_batchnorm=True)
     optimizer = optim.Adam(model.parameters(), lr=0.001)
     train(model, X_train, Y_train, optimizer, steps=100)
-    assert evaluate(model, X_test, Y_test) > 0.7 # TODO: batchnorm doesn't work!!!
```
Comment (geohot) [inline test/models/test_mnist.py:103]: Wait,  so what's the error now? The assert is still 0.7

**Comment by geohot** [comment]
Turns out I wasn't lying, it does work now!

https://github.com/geohot/tinygrad/commit/623fb1ef281131ed0b3fd65111bc151a0717b098

**Pull request closed without merging**